SPEC.md

```diff
@@ -127,3 +128,19 @@ $ ipfs files mv /cohosting/lazy/ipfs.io /cohosting/full/ipfs.io
 ```
 
 Or the other way around.
+
+### Prune
+
+The act of deleting old snapshots. To remove all snapshots for a certain domain:
+
+```console
+$ ipfs files rm -r /cohosting/<lazy|full>/ipfs.io
+```
+
+To remove all but the last _n_ snapshots:
+
+```console
+$ ipfs files ls /cohosting/<lazy|full>/ipfs.io | tail -r | tail -n +3
+# For each snapshot
+$ ipfs files ls /cohosting/<lazy|full>/ipfs.io/$snapshot
+```
```

Comment:
Let's make it more clear and platform-agnostic (`tail -r` is not valid on Linux):

```bash
# assuming we have 3 snapshots:
$ ipfs files ls /cohosting/full/docs.ipfs.io | sort
2019-10-10_000001
2019-10-11_000002
2019-10-12_000003

# we decide to keep only the last (latest) one (n=1)
# 1. get snapshots ready to be pruned:
$ ipfs files ls /cohosting/full/docs.ipfs.io | sort | head -n -1
2019-10-10_000001
2019-10-11_000002

# 2. perform pruning by removing each snapshot:
$ ipfs files rm -r /cohosting/full/ipfs.io/2019-10-10_000001
$ ipfs files rm -r /cohosting/full/ipfs.io/2019-10-11_000002
```

Reply:
THANKS! I was actually going to ask you help on this one, but then I found out about `tail -r` and thought it would work everywhere!